### PR TITLE
Optuna ReadtheDocs top page addition

### PR DESCRIPTION
**Pull request by Crissman**
Adding more details to the top page of Optuna's documentation by converting some material from the README.md

### CODE REVIEW

**Comment by keisuke-umezawa** [comment]
fyi: generated pages https://34665-122299416-gh.circle-artifacts.com/0/docs/build/html/index.html

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,9 +1,127 @@
...
+   with developers.
+-  `Stack
+   Overflow <https://stackoverflow.com/questions/tagged/optuna>`__ for
+   questions.
+
+Contribution
+------------
+
+Any contributions to Optuna are welcome! When you send a pull request,
+please follow the `contribution guide <./CONTRIBUTING.md>`__.
```
Comment (keisuke-umezawa) [inline docs/source/index.rst:87]: I could not open the url.

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,9 +1,127 @@
...
+
+License
+-------
+
+MIT License (see `LICENSE <./LICENSE>`__).
```
Comment (keisuke-umezawa) [inline docs/source/index.rst:92]: I could not open the url.

File: docs/source/index.rst [hunk 3]
```diff
@@ -1,9 +1,127 @@
...
+
+.. |Python| image:: https://img.shields.io/badge/python-3.5%20%7C%203.6%20%7C%203.7%20%7C%203.8-blue
+   :target: https://www.python.org
+.. |pypi| image:: https://img.shields.io/pypi/v/optuna.svg
+   :target: https://pypi.python.org/pypi/optuna
+.. |conda| image:: https://img.shields.io/conda/vn/conda-forge/optuna.svg
+   :target: https://anaconda.org/conda-forge/optuna
+.. |GitHub license| image:: https://img.shields.io/badge/license-MIT-blue.svg
+   :target: https://github.com/optuna/optuna
+.. |CircleCI| image:: https://circleci.com/gh/optuna/optuna.svg?style=svg
```
Comment (keisuke-umezawa) [inline docs/source/index.rst:109]: It is not used.

File: docs/source/index.rst [hunk 4]
```diff
@@ -1,9 +1,127 @@
...
+   :target: https://circleci.com/gh/optuna/optuna
+.. |Read the Docs| image:: https://readthedocs.org/projects/optuna/badge/?version=stable
```
Comment (keisuke-umezawa) [inline docs/source/index.rst:111]: It is not used.

File: docs/source/index.rst [hunk 5]
```diff
@@ -1,9 +1,127 @@
...
+   :target: https://optuna.readthedocs.io/en/stable/
+.. |Codecov| image:: https://codecov.io/gh/optuna/optuna/branch/master/graph/badge.svg
```
Comment (keisuke-umezawa) [inline docs/source/index.rst:113]: It is not used.

File: docs/source/index.rst [hunk 6]
```diff
@@ -1,9 +1,127 @@
...
+   :target: https://codecov.io/gh/optuna/optuna/branch/master
+.. |Gitter chat| image:: https://badges.gitter.im/optuna/gitter.svg
```
Comment (keisuke-umezawa) [inline docs/source/index.rst:115]: It is not used.

File: docs/source/index.rst [hunk 7]
```diff
@@ -1,9 +1,127 @@
...
+Koyama. 2019. Optuna: A Next-generation Hyperparameter Optimization
+Framework. In KDD (`arXiv <https://arxiv.org/abs/1907.10902>`__).
+
+.. |Python| image:: https://img.shields.io/badge/python-3.5%20%7C%203.6%20%7C%203.7%20%7C%203.8-blue
+   :target: https://www.python.org
+.. |pypi| image:: https://img.shields.io/pypi/v/optuna.svg
+   :target: https://pypi.python.org/pypi/optuna
+.. |conda| image:: https://img.shields.io/conda/vn/conda-forge/optuna.svg
+   :target: https://anaconda.org/conda-forge/optuna
+.. |GitHub license| image:: https://img.shields.io/badge/license-MIT-blue.svg
```
Comment (keisuke-umezawa) [inline docs/source/index.rst:107]: It is not used.

File: docs/source/index.rst [hunk 8]
```diff
@@ -1,9 +1,127 @@
...
+
+Takuya Akiba, Shotaro Sano, Toshihiko Yanase, Takeru Ohta, and Masanori
+Koyama. 2019. Optuna: A Next-generation Hyperparameter Optimization
+Framework. In KDD (`arXiv <https://arxiv.org/abs/1907.10902>`__).
+
+.. |Python| image:: https://img.shields.io/badge/python-3.5%20%7C%203.6%20%7C%203.7%20%7C%203.8-blue
+   :target: https://www.python.org
+.. |pypi| image:: https://img.shields.io/pypi/v/optuna.svg
+   :target: https://pypi.python.org/pypi/optuna
+.. |conda| image:: https://img.shields.io/conda/vn/conda-forge/optuna.svg
```
Comment (keisuke-umezawa) [inline docs/source/index.rst:105]: It is not used.

File: docs/source/index.rst [hunk 9]
```diff
@@ -1,9 +1,127 @@
...
+Reference
+---------
+
+Takuya Akiba, Shotaro Sano, Toshihiko Yanase, Takeru Ohta, and Masanori
+Koyama. 2019. Optuna: A Next-generation Hyperparameter Optimization
+Framework. In KDD (`arXiv <https://arxiv.org/abs/1907.10902>`__).
+
+.. |Python| image:: https://img.shields.io/badge/python-3.5%20%7C%203.6%20%7C%203.7%20%7C%203.8-blue
+   :target: https://www.python.org
+.. |pypi| image:: https://img.shields.io/pypi/v/optuna.svg
```
Comment (keisuke-umezawa) [inline docs/source/index.rst:103]: It is not used.

File: docs/source/index.rst [hunk 10]
```diff
@@ -1,9 +1,127 @@
...
+MIT License (see `LICENSE <./LICENSE>`__).
+
+Reference
+---------
+
+Takuya Akiba, Shotaro Sano, Toshihiko Yanase, Takeru Ohta, and Masanori
+Koyama. 2019. Optuna: A Next-generation Hyperparameter Optimization
+Framework. In KDD (`arXiv <https://arxiv.org/abs/1907.10902>`__).
+
+.. |Python| image:: https://img.shields.io/badge/python-3.5%20%7C%203.6%20%7C%203.7%20%7C%203.8-blue
```
Comment (keisuke-umezawa) [inline docs/source/index.rst:101]: It is not used.

**Comment by Crissman** [comment]
Thanks, @keisuke-umezawa ! Cleaned up.

**Pull request merged**